### ISSUE TITLE
DAC 1 - High Priority WCAG Level A - Non-Text Content 01

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#1]
<img width="278" alt="DAC_Non_Text_Content_01" src="https://user-images.githubusercontent.com/75850106/155547435-671760bb-c8cc-403f-94c1-376683b00f83.png">

**Issue:** 
A map shows ‘Neighbourhoods by income deprivation’, and the map is solely visual. There is no content in the HTML that conveys which neighbourhoods are ‘More deprived’ or ‘Less deprived’ to screen reader users, meaning that screen reader users may be missing out on vital content. There are multiple maps on the page that are similar, and do not have a non- text alternative. 

**Suggested Solution:** 
A text alternative of the map must be present that conveys the maps content to users that are unable to visually see the map. This could be achieved by providing a table. 

**Additional information**

<img width="610" alt="DAC 1 Current code ref" src="https://user-images.githubusercontent.com/75850106/155547662-4cdffe25-c164-44c8-93a0-4ba16a25998c.png">

_Screen reader comments:_

“I selected ‘Allerdale’ from the first dropdown labelled ‘Select an area’ and examined the updated content. I encountered the text ‘This is shown in the first two bars in the chart on the top right in red’.
This was one of several highly visual references which were not helpful to me as a screen reader user. I subsequently confirmed that these references were present irrespective of the selection in the drop down.
Whilst I understand that much of the material presented may be graphical in nature, it would have been useful to provide a link to a textual description of the chart that was being referred to.”
